### PR TITLE
feat(config.js): add 'websocket' config option

### PR DESCRIPTION
**Pull request by paweldomas**
Config.js will allow to specify both BOSH and Websocket URLs(even at the same time). In such case the web app will prefer Websocket over BOSH. The reason is that it appears to be more stable and a bit faster on web, while on mobile Websocket is dropped too quickly(killed by the OS) on network changes.

### CODE REVIEW

**Comment by damencho** [comment]
Would you mind adding config in the template nginx config like the bosh one:
https://github.com/jitsi/jitsi-meet/blob/master/doc/debian/jitsi-meet/jitsi-meet.example#L56
And if a subdomain one is needed like this:
https://github.com/jitsi/jitsi-meet/blob/master/doc/debian/jitsi-meet/jitsi-meet.example#L86

**Comment by paweldomas** [comment]
> Would you mind adding config in the template nginx config like the bosh one:
https://github.com/jitsi/jitsi-meet/blob/master/doc/debian/jitsi-meet/jitsi-meet.example#L56
And if a subdomain one is needed like this:
https://github.com/jitsi/jitsi-meet/blob/master/doc/debian/jitsi-meet/jitsi-meet.example#L86

Yes, I need to look more into those.

**Comment by damencho** [comment]
I suppose @aaronkvanmeerten can help :)

File: config.js [hunk 1]
```diff
@@ -30,6 +30,9 @@ var config = {
     // BOSH URL. FIXME: use XEP-0156 to discover it.
     bosh: '//jitsi-meet.example.com/http-bind',
 
+    // Websocket URL
+    // websocket: 'wss://jitsi-meet.example.com/xmpp-websocket',
```
Comment (aaronkvanmeerten) [inline config.js]: We may eventually want to have an example comment for multidomain support like so:

    bosh: '///http-bind',
        websocket: 'wss:///xmpp-websocket',

Comment (paweldomas) [reply config.js]: on second thought,  it is in the nginx example and config.js is more as a doc/reference, so I will skip that if you're ok with it? just would rather not pollute it too much

Comment (aaronkvanmeerten) [reply config.js]: We can do more in the examples directory instead

Comment (paweldomas) [reply config.js]: it is kind of crazy how many examples are there...

Comment (damencho) [reply config.js]: We were talking with Saul in the next PR to delete that example filder

**Pull request merged**